pypeit/spectrographs/keck_deimos.py

```diff
@@ -167,6 +169,10 @@ def default_pypeit_par(self):
         par['calibrations']['wavelengths']['n_first'] = 3
         par['calibrations']['wavelengths']['match_toler'] = 2.5
 
+        # Do not require bias frames
+        turn_off = dict(use_biasimage=False)
+        par.reset_all_processimages_par(**turn_off)
```

Comment:
Nit picking: It might have been me that suggested we do it this way, but can you just do it on one line?  I.e., `par.reset_all_processimages_par(use_biasimage=False)`.